src/Microsoft.ML.TensorFlow/Microsoft.ML.TensorFlow.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="System.IO.FileSystem.AccessControl" Version="$(SystemIOFileSystemAccessControl)" />
     <PackageReference Include="System.Security.Principal.Windows" Version="$(SystemSecurityPrincipalWindows)" />
     <PackageReference Include="TensorFlow.NET" Version="$(TensorflowDotNETVersion)" />
+    <PackageReference Include="Google.Protobuf" Version="$(GoogleProtobufPackageVersion)" />
```

Comment:
>GoogleProtobufPackageVersion [](http://example.com/codeflow?start=59&length=28)

QQ: where this property is defined?

Reply:
Its in the `Versions.Props` file. We try to keep all our versions there.